pypom/page.py

```diff
@@ -8,18 +8,48 @@
 class Page(WebView):
 
     _url = None
+    """
+    A string which when appended to ``base_url``
+    indicates the url to the given page.
+
+    Defaults to ``None``.
+    """
 
     def open(self):
+        """
+        Navigates to the url returned by the :py:func:`url` property and
+        waits for the page to load by calling :py:func:`wait_for_page_to_load`.
+
+        :returns:
+            The current page object (i.e., ``self``).
+        """
         self.selenium.get(self.url)
         self.wait_for_page_to_load()
         return self
 
     @property
     def url(self):
```

Comment:
Not really related to your patch, but I'm questioning if `url` should return the actual URL (not that I'd say that's a necessary thing to return) rather than the constructed assumed URL of the page. I at least think this is unclear to consumers of PyPOM.